lib/maintainers.nix

```diff
@@ -9,6 +9,7 @@
   aforemny = "Alexander Foremny <alexanderforemny@googlemail.com>";
   aflatter = "Alexander Flatter <flatter@fastmail.fm>";
   aherrmann = "Andreas Herrmann <andreash87@gmx.ch>";
+  ajp = "Anders Papitto <anderspapitto@gmail.com>";
```

Comment:
It's very helpful when the maintainer attrname is the same as github username. One can then easily ping the maintainer by @anderspapitto.